lib/ansible/module_utils/k8s/common.py

```diff
@@ -62,6 +63,9 @@
         'type': 'bool',
         'default': False,
     },
+    'merge_type': {
```

Comment:
I think this will need to move out of the common arg spec, since it only applies to the k8s module (not scale or probably any future modules). You could implement the argspec property in raw.py, and just add the `merge_type` parameter there

Reply:
alternatively, you can just pop the key off in the `scale.py` implementation of argspec, it seems to be doing that already.